web-client/src/global.scss

```
@@ -24,3 +24,10 @@
 @import "~@ionic/angular/css/text-alignment.css";
 @import "~@ionic/angular/css/text-transformation.css";
 @import "~@ionic/angular/css/flex-utils.css";
+
+ion-toolbar,
+ion-content,
+ion-footer {
+  --background: var(--ion-color-prussian-blue);
+  --color: var(--ion-color-white);
+}
```

This creates white margins on either side of the app: [see UI change](https://www.chromatic.com/pullrequest?appId=60ddb6c36599fd003b211fd8&number=23&view=changes)

Rather than targeting these individual Ionic components, we should probably just set Ionic's global style variables, which should take effect across all components, like this?

```suggestion
:root {
  // Docs: https://ionicframework.com/docs/theming/themes#application-colors
  --ion-background-color: var(--ion-color-prussian-blue);
  --ion-background-color-rgb: var(--ion-color-prussian-blue-rgb);
  --ion-text-color: var(--ion-color-white);
  --ion-text-color-rgb: var(--ion-color-white-rgb);
}
```